packages/components/src/components/files/models.ts

```diff
@@ -60,6 +65,11 @@ export const DEFAULT_FILE: IFile = {
     name: undefined,
     options: undefined,
     propertiesRowId: undefined,
+    canDelete: undefined,
+    canEdit: undefined,
+    canRead: undefined,
+    canRename: undefined,
+    canUpload: undefined,
```

Comment:
Should these all default to `false` here?

Reply:
Yes!